### PR TITLE
Add namcap

### PR DESCRIPTION
**Pull request by lazka**
This is similar to saneman in pactoys, but used by Arch and somewhat maintained.

### CODE REVIEW

**Comment by lazka** [comment]
Some of the rules probably need adjustment for MSYS2 differences and to support DLLs, but this is a start.

File: namcap/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        '002-force-msys-python.patch'
+        'namcap.sh')
+sha256sums=('d40479dd533e1171987579c5e11a09d55628addaf891256389839e1701117e06'
+            'b3251592be11fcc7a667531c2b84bc1bbce25e29c7546d54b99a509b336dc429'
+            'e45456455412eee2838069cfb0079925dddc997e0cc887f66899c72ac1d93aad'
+            '875e4ddd70ad8f2cc26cdf9512f18872a76e8f5496e347b3a2701fc4f3fec0aa')
+noextract=(${pkgname}-${pkgver}.tar.gz)
+
+prepare() {
+  tar -xzvf ${pkgname}-${pkgver}.tar.gz -C ${srcdir} || true
```
Comment (elieux) [inline namcap/PKGBUILD:25]: Why `-C ${srcdir}`? Every function starts in `$srcdir` already. :)

**Comment by elieux** [comment]
LGTM. I should have some time to test it in action soon.

**Comment by lazka** [comment]
I've also filed an upstream bug re missing program exit codes: https://bugs.archlinux.org/task/66819

**Comment by elieux** [comment]
> I've also filed an upstream bug re missing program exit codes: https://bugs.archlinux.org/task/66819

Closed as ENOTABUG, see https://bugs.archlinux.org/task/60333.

Maybe namcap won't be of such great help as I'd hoped. We'll see.

**Comment by lazka** [comment]
oh well...

**Pull request merged**